cli.js

```diff
@@ -14,6 +14,7 @@ const cli = meow(`
 
 	Options
 	  --upload, -u  Measure upload speed in addition to download speed
+	  --oneline, -l  Reduce spacing and output on a single line
```

Comment:
I would prefer `--minimal` as a flag name. I also don't see the point of a short-flag as this flag is not something you would type out directly, but instead have in a script.

Reply:
You also need to update the readme output.

Reply:
Resolved with new flag name.